README.md

```diff
@@ -49,11 +49,6 @@ There are some FreshRSS extensions out there, developed by community members:
 * [Reddit Image](https://github.com/aledeg/FreshRSS-extensions/tree/master/xExtension-RedditImage): Replace link to Reddit topic with resource link
 
 
-### By [@Lapineige](https://github.com/lapineige)
-
-* [Reading Time](https://framagit.org/Lapineige/FreshRSS_Extension-ReadingTime): Add a reading time estimation next to each article.
```

Comment:
It could be good to keep an entry in the readme, even though the extension is in this repository.
Maybe make a new section higher up

Reply:
Well my goal initially was to point outside of github on purpose. But it was also the "source of truth" for the code, which is not updated to your last changes anymore, so I would understand if you prefer the local link.